### DISCUSSION TITLE
Two warnings appear while uploading the application to the store

### DISCUSSION THREAD

**Discussion by avcr2019** [airsdk/Adobe-Runtime-Support#3019] [General]
Welcome back
This problem or this warning has been appearing for me for more than a year and I ignore it every time I update the application. Is there a solution to these two warnings?
Especially since Google Console began to focus on them
![Screenshot 2024-01-20 at 2 31 52 PM](https://github.com/airsdk/Adobe-Runtime-Support/assets/51138931/46732c6f-4b64-4e39-8a97-f0834edb215a)

**Comment by ajwfrost** [comment]
Hi

I'm not sure there's much we can do about the second one - the native debug symbols we keep track of here, to help analyze crashes, but we're not currently allowed to distribute these (I believe the reasoning being, if we provided these then it would make it easier for bad folk to work out how to create security exploits for the Flash Player .. and whilst this reasoning is a little out of date now, it's hard to get the rules changed! - but I will try again)

For the first one, we don't enable proguard in the builds, so I'm not sure this is relevant or why they're showing it!

thanks

**Reply by avcr2019** [reply]
Hello
Thank you, my dear friend, for this information and for the response
Well, I will leave it as it is and will not do anything because Google changes its policy from time to time, so I will not do anything, and when the problem appears in every update, I will make a skip as I do every time.

Thanks
